README.md

```diff
@@ -31,6 +31,10 @@
 - [Ongoing Support](#ongoing-support)
 - [License](#license)
 
+**⚠️ Warning ⚠️**
+
+> The project has recently been renamed from QHub to Nebari. If your deployment is is still managed by `qhub`, performing an inplace upgrade will **IRREVOCABLY BREAK** your deployment. This will cause you to lose any data stored on the platform, including but not limited to, NFS (filesystem) data, conda-store environments, Keycloak users and groups, etc. Please [backup](https://docs.qhub.dev/en/latest/source/admin_guide/backup.html) your data before attempting an upgrade.
```

Comment:
```suggestion
> The project has recently been renamed from QHub to Nebari. If your deployment is is still managed by `qhub`, performing an inplace upgrade will **IRREVOCABLY BREAK** your deployment. This will cause you to lose any data stored on the platform, including but not limited to, NFS (filesystem) data, conda-store environments, Keycloak users and groups, etc. Please [backup](https://www.nebari.dev/docs/how-tos/manual-backup) your data before attempting an upgrade.
```